docs/reference/indices/analyze.asciidoc

```diff
@@ -96,6 +96,24 @@ coming[2.0.0, body based parameters were added in 2.0.0]
 Will cause the analysis to happen based on the analyzer configured in the
 mapping for `obj1.field1` (and if not, the default index analyzer).
 
+If `detail` parameter is `true`, this API output each tokens within tokenizer chain and output with all token attributes.
+It also output only token attribute specified by `attributes`.
+If `short_attr_name` is `false`, it output token attribute full name.
+
+[source,js]
+--------------------------------------------------
+curl -XGET 'localhost:9200/test/_analyze' -d '
+{
+  "field" : "obj1.field1",
+  "text" : "this is a test",
+  "detail" : true,
+  "attributes" : ["KeywordAttribute"],
+  "short_attr_name" : false
+}'
+--------------------------------------------------
+
+coming[2.0.0, body based parameters were added in 2.0.0]
```

Comment:
I think this is related the example of request. We add "body based param" style in 2.0.0.
Then, I add after the example.
If it doesn't need, I will remove this line.